core/iwasm/common/wasm_runtime_common.c

```diff
@@ -1891,6 +1894,67 @@ wasm_runtime_set_module_inst(WASMExecEnv *exec_env,
     wasm_exec_env_set_module_inst(exec_env, module_inst);
 }
 
+bool
+wasm_runtime_get_export_global_inst(const wasm_module_inst_t module_inst,
```

Comment:
Had better use `WASMModuleInstanceCommon *const module_inst` like other places.

Reply:
I just committed a fix for this.